postgres/datadog_checks/postgres/postgres.py

```diff
@@ -387,20 +385,25 @@ def _connect(self):
         else:
             self.db = self._new_connection(self._config.dbname)
 
-    # Reload the track_activity_query_size setting on a new connection to the main db
-    def _load_query_max_text_size(self, db):
+    # Reload pg_settings on a new connection to the main db
+    def _load_pg_settings(self, db):
         try:
             with db.cursor(cursor_factory=psycopg2.extras.DictCursor) as cursor:
-                self.log.debug("Running query [%s]", TRACK_ACTIVITY_QUERY_SIZE_QUERY)
-                cursor.execute(TRACK_ACTIVITY_QUERY_SIZE_QUERY)
-                row = cursor.fetchone()
-                self._db_configured_track_activity_query_size = int(row['setting'])
-        except (psycopg2.DatabaseError, psycopg2.OperationalError) as e:
-            self.log.warning("cannot read track_activity_query_size from pg_settings: %s", repr(e))
+                self.log.debug("Running query [%s]", PG_SETTINGS_QUERY)
+                cursor.execute(
+                    PG_SETTINGS_QUERY,
+                    ("pg_stat_statements.max", "track_activity_query_size"),
+                )
+                rows = cursor.fetchall()
```

Comment:
Nit: should re-create `self.pg_settings` here to ensure we throw away the old values when they're being reloaded